Yoann-Abbes/The-count-of-money
# add logs to bootstrap and add cronFetcher script

## What does this PR do?

This PR add `cronFetcher.js` script that fetch for each crypto currency referenced in crypto_list table:
  - the last minut every minut and delete the oldest
  - the last hour every hour and delete the oldest
  - the last day every day and delete the oldest

### How should this be manually tested?

You'll need the api key for cryptocompare api (teams).
You can set it in API_KEY env variable or with the `--api_key blablabla` parameter.

  - `cd backend`
  - `npm i`
  - `cd scripts`
  - `./init-db.sh`
  - `node bootstrap.js` (you can see the beautiful logs... :smile:)
  - `node cronFetcher.js`
  - Logs will tells you when new rows are add/removed from tables
  - You can verify the changes from your postgres console (`psql -h localhost -U postgres`)

### Other changes
This PR add logs with progress bars to bootstrap script.

### Boyscout
/
